app/javascript/lib/scp-api.js

```diff
@@ -5,7 +5,7 @@
  *
  * API docs: https://singlecell.broadinstitute.org/single_cell/api
  */
-
+import React from 'react'
```

Comment:
```suggestion
```

Is this left over from before modularizing `MessageModal`?

Reply:
we need this to be able to pass JSX into showMessage